packages/openmrs/test/index.test.js

```diff
@@ -66,65 +66,43 @@ describe('execute', () => {
 });
 
 describe('http', () => {
-  it('should GET with a query', async () => {
+  beforeEach(()=> {
+    // basic DELETE request interceptor
     testServer
       .intercept({
         path: '/ws/rest/v1/patient',
-        query: { q: 'Sarah 1' },
-        method: 'GET',
-      })
-      .reply(200, { results: [{ display: 'Sarah 1' }] }, { ...jsonHeaders });
-    const state = { configuration };
-
-    const { data } = await http.request('GET', '/ws/rest/v1/patient', {
-      query: { q: 'Sarah 1' },
-    })(state);
-
-    expect(data.results[0].display).to.eql('Sarah 1');
-  });
+        method: 'DELETE'
+    })
+      .reply(200, { results: 'Successful' }, {...jsonHeaders});
 
-  it('should auto-fetch patients with a limit', async () => {
```

Comment:
Have we dropped this test entirely? Why? It's testing some quite important behaviour